tests/unit/components/ContentNode/TabNavigator.spec.js

```diff
@@ -69,4 +69,49 @@ describe('TabNavigator.spec', () => {
     expect(wrapper.find('.tabs-content').text()).toBe('Second');
     expect(tabnav.props('value')).toEqual(titles[1]);
   });
+
+  it('selects correct tab when adding a tab', async () => {
+    const longerTitles = ['Long tab title',
+      'A Longer tab title', 'The Longest tab title', 'added title'];
+
+    const wrapper = createWrapper();
+    wrapper.setProps({ titles: longerTitles });
+
+    expect(wrapper.vm.currentTitle).toEqual('added title');
```

Comment:
Can we check that the right tab gets rendered as the current one instead of doing assertions on the actual view model instance?

Reply:
I think we already have a test that tests whether the content changes based on which tab is picked. Do you think it's still necessary to test whether the v-model instance works in getting the correct tab rendered?